sklearn_numba_dpex/kmeans/drivers.py

```diff
@@ -241,6 +240,9 @@ def _lloyd(
             n_samples,
             n_features,
             n_clusters,
+            with_sample_weight=sample_weight is not None,
+            return_inertia=bool(verbose),
+            compute_exact_inertia=True,
```

Comment:
Based on my other comment, I think `compute_exact_inertia` should not be exposed but be defined internally based on the value of `with_sample_weight` and `return_inertia`.